docs/book/supported_features_matrix.md

```diff
@@ -6,22 +6,24 @@ The latest vSphere CSI Driver release is recommended for better stability and pe
 |---------------------------------------------------------------------------------------------|-------------------------------------------|---------------|---------------|-------------|---------------|
-| Enhanced Object Health in UI for vSAN Datastores                                            | v1.0.3 to v2.2.1                          | ✅             | ✅             | ✅           | ✅             |
-| Dynamic Block PV support (`Read-Write-Once` Access Mode)                                    | v1.0.3 to v2.2.1                          | ✅             | ✅             | ✅           | ✅             |
-| Dynamic Virtual Volume (vVOL) PV support                                                    | v1.0.3 to v2.2.1                          | ✅             | ✅             | ✅           | ✅             |
-| Topology/Availability Zone support (beta) (Block Volume only)                               | v1.0.3 to v2.2.1                          | ✅             | ✅             | ✅           | ✅             |
-| Static PV Provisioning                                                                      | v1.0.3 to v2.2.1                          | ✅             | ✅             | ✅           | ✅             |
-| K8s Multi-node Control Plane support                                                        | v1.0.3 to v2.2.1                          | ✅             | ✅             | ✅           | ✅             |
-| [WaitForFirstConsumer](https://kubernetes.io/docs/concepts/storage/storage-classes/) (beta) | v1.0.3 to v2.2.1                          | ✅             | ✅             | ✅           | ✅             |
-| Offline Volume Expansion support (beta) (Block Volume only)                                 | v2.0.0 to v2.2.1                          | ✅             | ✅             | ✅           | ❌             |
-| Encryption support via VMcrypt (Block Volume only)                                          | v2.0.0 to v2.2.1                          | ✅             | ✅             | ✅           | ❌             |
-| Dynamic File PV support through vSAN 7.0 File Services on vSAN Datastores                   | v2.0.0 to v2.2.1                          | ✅             | ✅             | ✅           | ❌             |
-| In-tree vSphere volume migration to CSI (beta)                                              | v2.1.0 to v2.2.1                          | ✅             | ✅             | ❌           | ❌             |
-| Online Volume Expansion support (beta) (Block Volume only)                                  | v2.2.0 to v2.2.1                          | ✅             | ❌             | ❌           | ❌             |
+| Enhanced Object Health in UI for vSAN Datastores                                            | v1.0.3 to v2.3.0                          | ✅             | ✅             | ✅           | ✅             |
```

Comment:
since 1.0.3 is deprecated, let's we start with the supported versions rather

Reply:
done